docs/migration.md

```diff
@@ -0,0 +1,57 @@
+# Migration
+
+Existing servers can migrate from one database backend to another. This page includes guidance for migrating from sqlite - similar concepts apply when migrating from databases of other types.
+
+NOTE: [pgloader](https://github.com/dimitri/pgloader) is a tool that can easily migrate to postgres from other databases. Consider it if your target database is postgres
+
+The process is as follows:
+
+1. Create a dump of existing data.
+2. Change all `CREATE TABLE ...` lines to `DELETE FROM tablename;`. We will later have lldap create the schema for us, so we want to clear out existing data to replace it with the original data.
```

Comment:
And in that case, we don't have to start/stop LLDAP just to create the schema below.

Reply:
It's not super consistent. For postgres i know of the following conversions:

text(255) -> text
blob -> bytea
integer -> bigint

I went with this approach because it guarantees the generated schema is what sea-orm expects and likes. I thought this process was a lot less error prone. A nice alternative would maybe be making an lldap command to either create schema at a target database url or even copy the existing database to the target

Reply:
Yeah, after thinking about it, it's not ideal. Your approach makes sense.

I'm willing to take a PR adding a command to create the DB schema, that's a nice idea! It should be fairly easy to implement (it would actually create the DB and/or apply all the migrations), you can look at the "send test email" command for the wiring.

Reply:
We can update this doc later once we have this command.